bootstrap.sh

```diff
@@ -114,7 +122,7 @@ function build_osrm() {
     if [[ ${NM:-false} != false ]]; then
         CMAKE_EXTRA_ARGS="${CMAKE_EXTRA_ARGS} -DCMAKE_NM=${NM}"
     fi
-    ../../../mason_packages/.link/bin/cmake ../ -DCMAKE_INSTALL_PREFIX=${MASON_HOME} \
+    ${MASON_HOME}/bin/cmake ../ -DCMAKE_INSTALL_PREFIX=${MASON_HOME} \
```

Comment:
@daniel-j-h - thanks, ticketed at #172 (which is relevant to the -flto work with clang)